EIPS/EIP-X-add-token-uri-kylanhurt.md

```diff
@@ -0,0 +1,60 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: URL Format for Tokens
+    Author: Kylan Hurt <kylan@edgesecure.co>, Eliran Zach <eliran@edgesecure.co>
+    Type: Standard Track
+    Category: ERC
+    Status: Draft
+    Created: 2018-04-01
+    Requires: 20, 831
+
+## Simple Summary
+A standard way of representing token information, facilitating the process of adding ERC #20 tokens as URLs on the application layer.
+
+## Abstract
+URLs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages provide for robust cross-application signaling between very loosely coupled applications. A standardized URL format for fundamental token information allows for instant invocation of the user's preferred wallet application (even if it is a webapp or a swarm đapp), with the correct parameterization of the token's information only to be confirmed by the user.
+
+## Motivation
+The convenience of representing payment requests by standard URLs has been a significant factor in the cryptocurrency. Extending this convenient mechanism to support embedding of basic token information would help to facilitate the usage of Ethereum-based tokens. In particular, QR-code embedded URLs could be generated by token-generators to help users track the tokens via any wallet software that has the ability to scan QR codes (particularly mobile wallets). For example, rather than users needing to find the contract address and number of decimal places for a given token through an online directory, they could simply scan a QR-code provided by the token generator with this data embedded. The wallet software could then automatically add the tokens to the user's wallet. This would improve the user experience associated with purchasing and holding tokens as a digital asset.
+
+
+## Specification
+
+### Syntax
+Token data URLs contain "ethereum" in their schema (protocol) part and are constructed as follows:
+
+    request                 = "ethereum" ":" [ "token-" ]target_address [ "@" chain_id ] [ "?" parameters ]
```

Comment:
This seems too generic, and may already be in use. Could you pick something more token-specific, or else make sure this is compatible with other uses of that URI?

Reply:
token should not be optional here - for this ERC "token-" should be mandatory